roles/config-routes/tasks/route.yml

```diff
@@ -0,0 +1,10 @@
+---
+
+- name: "Configure route for the specific interface"
+  template:
+    src: route.j2 
+    dest: /etc/sysconfig/network-scripts/route-{{ route.device }}
+  with_items:
+  - '{{ routes }}'
+  loop_control:
+    loop_var: route
```

Comment:
I have the same concern.   Can you add a Debug statement stating that config is complete but a restart of the interface is needed to apply new static route(s).